test/integration/common_test.go

```diff
@@ -108,7 +108,7 @@ func authAndIssue(c *client, csrKey *ecdsa.PrivateKey, domains []string) (*issua
 	}
 	order, err := c.Client.NewOrder(c.Account, ids)
 	if err != nil {
-		return nil, fmt.Errorf("making order: %s", err)
```

Comment:
this is a place we should be using the fancy new error wrapping stuff in Go 1.13. I'm fixing a bug quickly instead of doing my assigned sprint tasks so I went with a smaller diff that still made it possible for me to inspect the returned problem at the cost of the "making order" context that is lost.